command/quota_status.go

```diff
@@ -171,17 +171,27 @@ func formatQuotaLimits(spec *api.QuotaSpec, usages map[string]*api.QuotaUsage) s
 			return used, ok
 		}
 
+		specBits := 0
+		if len(specLimit.RegionLimit.Networks) == 1 {
+			specBits = *specLimit.RegionLimit.Networks[0].MBits
+		}
+
 		used, ok := lookupUsage()
 		if !ok {
 			cpu := fmt.Sprintf("- / %s", formatQuotaLimitInt(specLimit.RegionLimit.CPU))
 			memory := fmt.Sprintf("- / %s", formatQuotaLimitInt(specLimit.RegionLimit.MemoryMB))
-			limits[i] = fmt.Sprintf("%s|%s|%s", specLimit.Region, cpu, memory)
+			net := fmt.Sprintf("- / %s", formatQuotaLimitInt(&specBits))
```

Comment:
looks like  `formatQuotaLimitInt` prints "inf" if specBits is zero, noting this for anyone else. Seems right to me since that means "no limit"